index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="icon" type="image/x-icon" href="./images/favicon-32x32.png">
+    <link href="https://fonts.googleapis.com/css2?family=Fraunces:opsz,wght@9..144,700&family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,200;1,400;1,600;1,900&display=swap" rel="stylesheet">
+    <title>Product preview card</title>
+</head>
+<body>
+    <div class="cardContainer">
+        <div class="imageContainer"></div>
+        <div class="infoContainer">
+            <p class="productType">P E R F U M E</p>
+            <h1>Gabrielle Essence Eau De Parfum</h1>
+            <p>A floral, solar and voluptuous interpretation composed by Olivier Polge, Perfumer-Creator for the House of CHANEL.</p>
+            <div class="pricesContainer"><p class="presentPrice">$149.99</p><p class="previousPrice">$169.99</p></div>
```

Comment:
ta nowa i stara cena jest jak najbardziej w porządku, ale można iść krok dalej i wykorzystać znaczniki semantyczne, które przydają się w takich przypadkach - czyli usunięcie jakiegoś tekstu i wskazanie na tekst, który został dodany w jego miejsce - są to 'del' i 'ins':
https://www.w3schools.com/tags/tag_del.asp

Reply:
i wtedy byłoby one zamiast 'p'